server/main/static/main/login.css

```diff
@@ -90,3 +90,9 @@ a.forgotPassword {
     color: red;
     font-size: 12px;
 }
+
+.alert h2 {
+    font-size: 1em;
```

Comment:
Per discussion in #49, should avoid messing with font sizing until an upgrade from Bootstrap 3.3 to 5.x+ can happen. I think you can get a similar result by just using a normal `<P>` for the error and using `role="alert"` on the parent element.

Reply:
I remove the font-size rule and split up how this works with a screen reader vs. without: there's an `sr-only` above the "Skip to Main Content" link, so the error is announced directly after the page title. The visible alert is `aria-hidden` to avoid being announced twice.